src/databricks/labs/ucx/install.py

```diff
@@ -212,10 +220,30 @@ def _new_wheel_builder(self):
     def _new_sql_backend(self, config: WorkspaceConfig) -> SqlBackend:
         return StatementExecutionBackend(self._ws, config.warehouse_id)
 
+    def _confirm_force_install(self) -> bool:
+        if os.environ.get('UCX_FORCE_INSTALL'):
```

Comment:
simplify? `if not os.environ.get('UCX_FORCE_INSTALL'): return False`. you probably want to introduce a variable, because this value is read three times.